spec/index.html

```diff
@@ -883,7 +879,7 @@ <h3>Overview</h3>
         <li>The length of <var>s</var> is less than or equal to the length
           of any other gossip path string <var>s′</var>.</li>
         <li>If <var>s</var> and <var>s′</var> have the same length (as strings),
-          then <var>s</var> is <a>lexicographically</a> less than or equal to <var>s′</var>.</li>
+          then <var>s</var> <a>code point orders</a> less than or equal to <var>s′</var>.</li>
```

Comment:
This reads a little funny to me ... should it be this?:

```suggestion
          then <var>s</var> is <a>code point ordered</a> less than or equal to <var>s′</var>.</li>
```